service.go

```diff
@@ -189,15 +187,18 @@ func prepArgusListenerClientConfig(cfg *ListenerConfig, watches ...Watch) {
 
 type ServiceIn struct {
 	fx.In
-	Config   Config
-	Listener ListenerConfig
-	Client   *http.Client
+	Config Config
+	Client *http.Client
 }
 
 func ProvideService() fx.Option {
 	return fx.Provide(
 		func(in ServiceIn) (*ClientService, error) {
 			svc, err := NewService(in.Config, getLogger)
+			if err != nil {
```

Comment:
why are we using errors.Join here?

Reply:
good question 🙂 , I added it to outlined the result to make debugging easier, i.e. `errFailedConfig = errors.New("ancla configuration error")`